packages/esm-commons-lib/src/components/encounter-list-tabs/encounter-list-tabs.component.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Tabs, Tab, TabList, TabPanels, TabPanel } from '@carbon/react';
+import { useConfig } from '@openmrs/esm-framework';
+import { EncounterList, getMenuItemTabConfiguration } from '@ohri/openmrs-esm-ohri-commons-lib';
+import styles from './encounter-list-tabs.scss';
+
+interface TabsComponentProps {
+  patientUuid: string;
+  configSchema: any;
+}
+
+export const TabsComponent: React.FC<TabsComponentProps> = ({ patientUuid, configSchema }) => {
+  const config = useConfig();
```

Comment:
This will not work because at this point the config is for the commons lib, we need to pass the config at the package/app level since the configs are different across packages.